sdk/core/Azure.Core.TestFramework/src/TestRecording.cs

```diff
@@ -33,17 +33,18 @@ private TestRecording(RecordedTestMode mode, string sessionFile, TestProxy proxy
             _recordedTestBase = recordedTestBase;
         }
 
-        public static async Task<TestRecording> CreateAsync(RecordedTestMode mode, string sessionFile, TestProxy proxy, RecordedTestBase recordedTestBase)
+        public static async Task<TestRecording> CreateAsync(RecordedTestMode mode, string sessionFile, TestProxy proxy, RecordedTestBase recordedTestBase, string assetsJson)
```

Comment:
The RecordedTestBase.AssetsJsonPath property is already publicly settable (though I'm not sure why that is), so we shouldn't need to update this method.